Packs/Base/ReleaseNotes/1_1_7.md

```diff
@@ -0,0 +1,9 @@
+
+#### Scripts
+##### SaneDocReports
+- Updates the sane-doc-reports docker image
+  - Fixes markdown html tags inconsistencies
+  - Fixes trend direction icons and edge case when there are large values
+  - Fixes markdown placeholder styles
+  - Fixes array instead of 0 when number is zero
+  - Fixes empty markdown pagebreak elements not working
```

Comment:
```suggestion
#### Scripts
##### SaneDocReports
- Updates the sane-doc-reports docker image
  - Fixes markdown HTML tag inconsistencies.
  - Fixes trend direction icons as well as edge cases when there are large values.
  - Fixes markdown placeholder styles.
  - Fixes an issue where an array would be returned instead of `0` when the number is zero.
  - Fixes empty markdown page break elements not working.
```